InstallGuide.md

```diff
@@ -0,0 +1,49 @@
+Canal provides two things
+
+- Networking between hosts in your cluster
+- Enforcement of network policy
+
+# Networking
+Networking can be handled either by `flannel` or by `calico`. Regardless of which is used, Calico needs to be involved when containers are started. See the [Policy](#policy) section below. 
+
+## Flannel
+If flannel is being used to provide inter-host connectivity then any of the flannel backends can be used. To set up flannel follow the [instructions](https://coreos.com/flannel/docs/latest/flannel-config.html).
+
+Flannel can run as a native binary or from inside a container.
```

Comment:
As someone following this guide, I'd like to see an example of at least one of these that I could mostly copy-and-paste to get working.

The instructions linked above are OK and we should keep that link, but is there a way we could have a "here's a docker command to start the flannel container with the settings required for canal".  

I guess I'm most interested in seeing some requirements here as for how flannel must be configured to run in canal.